apm-protocol/apm-network/src/main/java/org/apache/skywalking/apm/network/trace/component/ComponentsDefine.java

```diff
@@ -156,4 +156,6 @@
     public static final OfficialComponent UNDERTOW = new OfficialComponent(84, "Undertow");
 
     public static final OfficialComponent FINAGLE = new OfficialComponent(85, "Finagle");
+
+    public static final OfficialComponent MARIADB = new OfficialComponent(86, "Mariadb");
```

Comment:
I think in Java, this should be `Mariadb-jdbc` or something indicating we instrumenting the client driver only.